Feisty/AppDelegate.swift

```diff
@@ -40,6 +42,29 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
       
     }
     
+    stub(condition: isHost("store.steampowered.com") && isPath("/api/appdetails")) { request in
```

Comment:
So this approach for stubbing into a different environment is actually quite clever but I don't know how I feel about the code existing directly in AppDelegate.

Another idea to play around with in the future is having an Environment singleton and then just set that class before doing your calls which will revert to asking that class where to direct those requests. I had a short article on Wiremock where I used something similar:
https://medium.com/dvt-engineering/using-wiremock-in-api-driven-development-75eb2707b6b8

Just something to read up on as well